include/dfu/fmfu_fdev.h

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef FMFU_FDEV_H__
+#define FMFU_FDEV_H__
+
+#include <device.h>
+
+/**
+ * @defgroup fmfu_fdev Full Modem Firmware Update from flash device
+ * @{
+ *
+ * @brief Functions for applying a full modem firmware update from
+ *        flash device.
+ */
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief Load the modem firmware update from the external flash to the modem.
+ *
+ * The modem library must be initialized in DFU mode before calling this
```

Comment:
nrf_modem_lib or is modem library clear enough?

Reply:
I think this should suffice, that way we don't get outdated docs if the naming changes. I don't think refs will fail if they don't exist.